Singer.API/Controllers/EventController.cs

```diff
@@ -1,13 +1,24 @@
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
 using Singer.DTOs;
 using Singer.Models;
+using Singer.Services;
 using Singer.Services.Interfaces;
 
 namespace Singer.Controllers
 {
    public class EventController : DataControllerBase<Event, EventDTO, CreateEventDTO, UpdateEventDTO>
    {
+      private readonly IEventService _eventService;
+
       public EventController(IEventService eventService) : base(eventService)
       {
+         _eventService = eventService;
+      }
+      [HttpPost("search")]
```

Comment:
Shouldn't we be using GET for searching?

Reply:
Well, I wanted to use the body for the DTO, otherwise the datetime-stuff would become annoying again, especially when using it in the URL (and with optional search params). And the abstraction of the databaseservice would require me to do more work.
TLDR: yes, but no.

Reply:
agree, good points 👍